src/ui/components/Library/Team/View/TestRuns/Overview/TestResultListItem.tsx

```diff
@@ -7,23 +7,30 @@ import styles from "../../../../Library.module.css";
 
 function ViewReplay({ passed }: { passed: boolean }) {
   return (
-    <MaterialIcon
-      iconSize="2xl"
-      outlined
-      className={
-        passed
-          ? "text-green-500 group-hover:text-green-700"
-          : "text-red-500 group-hover:text-red-700"
-      }
-    >
-      play_circle
-    </MaterialIcon>
+    <Link href={`/recording/${recordingId}`}>
```

Comment:
This variable is undefined.

Reply:
Yeah, I'm not sure what happened but it appears to have been a messy merge/collision with Jaril code in the same hour?

I spent time trying to unwrap it and I'm a little confused about the state of things. I closed this PR before this comment but it's possible it's still causing trouble somewhere? If so a) sorry b) feel free to fix/roll back.

Reply:
Hm, not sure. The commit in this PR looks like it was a little broken (there was no `recordingId` variable) but at least it was an easy fix :)